src/FakeItEasy.Net35/Compatibility/ConcurrentQueue.cs

```diff
@@ -0,0 +1,34 @@
+namespace FakeItEasy

```

Comment:
It _is_ because it targets .NET 4.5. Yes, it should target 3.5, but we made the change in #568 in order to prepare for moving the tests to xUnit 2.0 (#545). We made the change in target a little earlier so it would be something that @mat-mcloughlin wouldn't have to worry about as he converted the test innards.